crates/viewer/re_recording_panel/src/recording_panel_ui.rs

```diff
@@ -406,6 +406,11 @@ fn dataset_entry_ui(
             ctx.command_sender()
                 .send_system(SystemCommand::CopyViewerUrl(url));
         }
+
+        if ui.button("Copy dataset name").clicked() {
+            re_log::info!("Copied {name:?} to clipboard");
+            ui.ctx().copy_text(name.clone());
+        }
```

Comment:
I literally run into this now that I needed to copy a dataset _id_ rather than name and I couldn't do it from the web build linked by this PR.
That risk making it cluttery, but can we please also add that here?